ckeditor/static/ckeditor/ckeditor-init.js

```diff
@@ -57,7 +57,8 @@
     document.body.addEventListener('click', function(e) {
       if (e.target && (
         e.target.matches('.add-row a') ||
-        e.target.matches('.grp-add-handler')
+        e.target.matches('.grp-add-handler') ||
+        e.target.parentElement.matches('.grp-add-handler')
```

Comment:
I think `.closest(".grp-add-handler")` might work well here instead. https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Reply:
Yes!

Reply:
I notice that there's a polyfill for `Element.prototype.matches` above, but `closest` has even less support (caniuse compare: [Element.closest()](https://caniuse.com/#feat=element-closest) vs [matches()](https://caniuse.com/#feat=matchesselector)). Notably, it's not available in any versions of IE.